cassovary-core/src/main/scala/com/twitter/cassovary/util/FastUtilUtils.scala

```diff
@@ -61,4 +61,6 @@ object FastUtilUtils {
 
   def newInt2IntOpenHashMap(): mutable.Map[Int, Int] =
     new Int2IntOpenHashMap().asInstanceOf[jutil.Map[Int, Int]]
+
+  def intArrayListToSeq(list: IntArrayList): Seq[Int] = list map { _.toInt }
```

Comment:
Good point about O(n)!  I've now changed it to
def intArrayListToSeq(list: IntArrayList): Seq[Int] = list.elements()
which should be O(1).  I couldn't find an immutable.Seq wrapper in the standard library.  This has the downside that technically a client could cast the result to WrappedArray[Int] and then mutate it, but clients who do that should expect mal-formed graphs